modules/test/playwright/tests/frontend-data-set-views-web/settings.spec.ts

```diff
@@ -222,12 +187,442 @@ test.describe('Data Set Settings', () => {
 					settingsPage.defaultVisualizationModeLabel
 				).toContainText('List');
 			});
-
-			await test.step('Clean data', async () => {
-				await dataSetManagerApiHelpers.deleteDataSet({
-					erc: customDataSetERC,
-				});
-			});
 		});
 	});
 });
+
+export const fragmentTest = mergeTests(
```

Comment:
I don't like having mixed tests for DSM and fragment in the same file, even if they usually test how the configuration of one is applied in the other.

With the growth of the API that allow us to create all needed data without using the UI we could split tests based on its usage, x.e. adding a prefix for the same functionality, like `dsmSettings.spec.ts` and `fragmentSettings.spec.ts`.

Any thoughts?

Reply:
Absolutely agree, and we could even create different folders for DSM and fragment ones